network-api/networkapi/wagtailpages/templatetags/bg_nav_tags.py

```diff
@@ -1,9 +1,17 @@
 from django import template
+from django.apps import apps
 from urllib.parse import urlparse
 
 register = template.Library()
 
 
+@register.simple_tag(name='get_bg_home_page')
```

Comment:
This is the new tag that we can use in templates whenever we want to access the PNI home page